src/panels/lovelace/components/hui-conditional-base.ts

```diff
@@ -132,14 +138,15 @@ export class HuiConditionalBase extends ReactiveElement {
     this._setVisibility(conditionMet);
   }
 
-  private _setVisibility(conditionMet: boolean) {
+  protected _setVisibility(conditionMet: boolean) {
```

Comment:
Should a protected function start with a `_`? 🤔

Reply:
Of course not ! 😅